components/admin_console/saml_settings.jsx

```diff
@@ -302,6 +310,19 @@ export default class SamlSettings extends AdminSettings {
             );
         }
 
+        let consumerServiceUrlHelp;
+        if (this.state.siteUrlSet) {
+            consumerServiceUrlHelp = (<FormattedMessage
```

Comment:
Do you mind changing these so that the component is on its own line when it spans multiple lines to have this consistent with other code? Like
```
consumerServiceUrlHelp = (
    <FormattedMessage
        ...
    />
);
```

I thought that was enforced by ESLint, but I guess not